### ISSUE TITLE
[Bug]: CREATE TABLE with STRICT raises syntax error

### ISSUE THREAD

**Issue by fehmud** [sqlitebrowser/sqlitebrowser#3744]
### What did you do?

Running *DB.Browser.for.SQLite-v3.13.0-x86.64.AppImage*.

I tried modifying a table via *Database Structure > Modify table* but got a syntax error message, so I clicked on *Create Table* to create a simple strict table but got the same error:

*Error creating table. Message from database engine:
near "STRICT": syntax error (CREATE TABLE "test" (
	"Field1"	INTEGER
) STRICT;)*

I switched to *Execute SQL* and ran the SQL from above:

*CREATE TABLE "test" (
	"Field1"	INT
) STRICT;*

and got the same error:

*Execution finished with errors.
Result: near "STRICT": syntax error
At line 1:
CREATE TABLE "test" (
	"Field1"	INT
) STRICT*

Tried again with a new database and got the same error.

### What did you expect to see?

Modification to strict table and creation of a strict table.

### What did you see instead?

A SQL syntax error.

### DB4S Version

3.13.0

### What OS are you seeing the problem on?

Linux

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by midoragh** [comment]
Check the SQLite version.
The STRICT keyword at the end of a CREATE TABLE statement is only recognized by SQLite version 3.37.0 (2021-11-27) and later.

**Comment by justinclift** [comment]
Oh, that sounds like there might be a packaging bug with the AppImage then.

@fehmud In theory the Help -> About dialog should show what version of SQLite it's trying to use.  What does that say? :smile:

**Comment by justinclift** [comment]
Yep, sounds like the AppImage is packaging the wrong SQLCipher library. :wink:

@lucydodo Any interest in taking a look?

**Comment by lucydodo** [comment]
I'll do some testing tomorrow and fix it. Note that this issue is due to the fact that we currently get the SQLCipher library from the official Ubuntu mirror, so we are picking up a relatively outdated version.

**Comment by lucydodo** [comment]
This issue was resolved with commit 4d530dce9db74acf57068b8179cecebbf85a972f.
Until a new stable version release, you can use either a continuous build or a nightly build:
- https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous
- https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/nightly

**Comment by mgrojo** [comment]
@lucydodo This solution might include support for the STRICT keyword, but then it breaks support on older distributions altogether. The [recommendation by AppImageKit](https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages/cc2441518975caca23e9ce2dba6f08a22c678d1e#libstdcso6) is to use the oldest still-supported LTS. Maybe we can still build with Ubuntu 20.04, but including a more modern version of SQLite than the one distributed with it.

This is what happens on Ubuntu 22.04:
```
https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages/cc2441518975caca23e9ce2dba6f08a22c678d1e#libstdcso6
```

**Comment by lucydodo** [comment]
Yes, in fact, I learned this a little while ago. I plan to patch it late this afternoon. :)
Related: 3781

**Comment by lucydodo** [comment]
Fixed via 5ff87937c16b82e02c2ba774e9df7373b996ad82
